main.js

```diff
@@ -52,4 +52,64 @@ for (let i = 0; i < arr10.length; i++) {
     if (arr10[i] % 2 === 0) {
         console.log(arr10[i]);
     }
+}
+
+//Lesson 16
+//task 16.5
+let str = 'шалаш';
+let str2 = 'Ирина';
+const  polyndrome = function (poly) {
+    let res = false;
+    for (let i = 0; i <= poly.length; i++) {
+        if (poly[i] !== poly[poly.length - (i + 1)]) {
+            res = false;
+            break;
```

Comment:
It doesn't matter as breaks on over the instructions anyway.

Reply:
ok.

Reply:
When I remove break, the program does not run correctly.

Reply:
If I remove `break`, the program does all the same.